platforms/hyperledger-fabric/configuration/roles/delete/vault_secrets/tasks/main.yaml

```diff
@@ -83,6 +83,8 @@
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/ca
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/orderers/{{orderer.name}}.{{ component_name }}/tls
     vault delete secret/crypto/ordererOrganizations/{{ component_name }}/orderers/{{orderer.name}}.{{ component_name }}/msp
+    vault delete secret/crypto/ordererOrganizations/{{ component_name }}/users/admin/tls
+    vault delete secret/crypto/ordererOrganizations/{{ component_name }}/users/admin/msp
```

Comment:
Add these to the docs as well
https://blockchain-automation-framework.readthedocs.io/en/latest/architectureref/certificates_path_list_fabric.html

Reply:
added